web/shared/src/typeahead.ts

```diff
@@ -222,6 +222,55 @@ export function get_emoji_matcher(query: string): (emoji: EmojiSuggestion) => bo
 // from BEFORE_MENTION_ALLOWED_REGEX in zerver/lib/mention.py later.
 export const word_boundary_chars = " _/-";
 
+export function triage_raw_with_multiple_items<T>(
```

Comment:
This is somewhat duplicated, but seemed a cleaner way to do this.

Reply:
This probably isn't how we'll want it to work in general, but we can merge this and fix forward.

I would kinda expect us to be able to make `triage` support this and just have existing `get_item` functions return a single-item array.